README.md

```diff
@@ -301,12 +304,18 @@ const observer = {
       console.log(`Device ID: ${mediaDeviceInfo.deviceId} Microphone: ${mediaDeviceInfo.label}`);
     });
   },
+
   audioOutputsChanged: freshAudioOutputDeviceList => {
     console.log('Audio outputs updated: ', freshAudioOutputDeviceList);
   },
+
   videoInputsChanged: freshVideoInputDeviceList => {
     console.log('Video inputs updated: ', freshVideoInputDeviceList);
-  }
+  },
+
+  audioInputMuteStateChanged: (device, muted) => {
+    console.log('Device', device, muted ? 'is muted in hardware' : 'is not muted);
```

Comment:
Just a question: in a realistic application, do builders want to compare this with their app mute state and show a notification if they don't match?

Reply:
If it's a realistic concern for their application, they will want to track this state and typically enter some error state — e.g., showing the mic selector greyed out with a "!" and some warning or similar. It's not a mute that we can un-mute, so it's closer to an error state that gets fixed outside of the app.

Reply:
Note particularly that `muted` is _not_ set by the app — that's `enabled`. `muted` means "this stream is broken and we think it's because the OS muted it".